config/environments/development.rb

```diff
@@ -49,7 +49,7 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
-  config.active_job.queue_adapter = :sidekiq
```

Comment:
Ryan, I don't know how you have your development environment set up. Is this change okay?

Reply:
I do typically use sidekiq locally.

But I'm okay with commenting this out with an additional comment like `used for testing Sidekiq locally`

Reply:
> I do typically use sidekiq locally.

In that case, I'll revert this change. I can get used to running sidekiq locally.